src/cpp/r/session/REmbeddedWin32.cpp

```diff
@@ -183,80 +183,6 @@ int askYesNoCancel(const char* question)
    }
 }
 
-bool initializeMaxMemoryDangerously()
```

Comment:
Were these all removed because they only run for R < 4.2? (Do we now require R 4.2+?)

Reply:
These are also normally initialized as part of `cmdlineoptions()`, but we stopped doing that with R 4.2 because of crashes we were seeing. https://github.com/rstudio/rstudio/issues/9824

Previously, we avoided using `cmdlineoptions()` because we were seeing an "invalid parameter" crash. However, we later learned that MinGW by default ignores "invalid parameter" errors, so we now resolve the issue by also setting a no-op "invalid parameter" handler.

For the curious:

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/set-invalid-parameter-handler-set-thread-local-invalid-parameter-handler?view=msvc-170

https://github.com/mirror/mingw-w64/blob/eff726c461e09f35eeaed125a3570fa5f807f02b/mingw-w64-crt/crt/crtexe.c#L261

https://github.com/mirror/mingw-w64/blob/eff726c461e09f35eeaed125a3570fa5f807f02b/mingw-w64-crt/crt/crtexe.c#L98-L108